.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -8,10 +8,17 @@ body:
     id: pre-check
     attributes:
       label: Search before asking
-      description: "Please search [issues](https://github.com/FasterXML/jackson-module-kotlin/issues) to check if your issue has already been reported."
+      description: |-
+        Please search [issues](https://github.com/FasterXML/jackson-module-kotlin/issues) to check if your issue has already been reported.
+        Also make sure that the problem occurs only in Kotlin.
```

Comment:
Could maybe reworded in a way to suggest it is best effort, as submitter may likely not know if it is Kotlin-specific or not. Perhaps the next sentence could be moved before this, to first explain KotlinModule is an extension of `jackson-databind`.

But beyond that, maybe something like:

"Please try to ensure that the problem occurs only in Kotlin, and not regular Java objects (POJOs)."

Reply:
Reflected.
However, `Kotlin` basically uses constructors as creator, whereas `POJO` should have included a constructor with no arguments in its definition, so it has been removed from the wording.
[4226e43](https://github.com/FasterXML/jackson-module-kotlin/pull/754/commits/4226e43c54c9115a8288d1e8227866a0afae8576)